releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md

```diff
@@ -1,6 +1,31 @@
 ## Microsoft.Extensions.Configuration.AzureAppConfiguration
 ### [Package (NuGet)](https://www.nuget.org/packages/Microsoft.Extensions.Configuration.AzureAppConfiguration)
 
+### 6.0.0 - November 29, 2022
+### Breaking Changes:
+* Removed `IConfigurationRefresher.SetDirty` API to migrate to `IConfigurationRefresher.ProcessPushNotification` API for necessary data consistency in push-model based configuration refresh. Refer to [this tutorial](https://docs.microsoft.com/en-us/azure/azure-app-configuration/enable-dynamic-configuration-dotnet-core-push-refresh?tabs=windowscommandprompt) for more details about the `ProcessPushNotification` API. [#357](https://github.com/Azure/AppConfiguration-DotnetProvider/issues/357)
+* Removed .NET 5 as a target framework as .NET 5 is out of support. [#391](https://github.com/Azure/AppConfiguration-DotnetProvider/pull/391)
+* Removed support for Feature Management V2 schema introduced in Microsoft.FeatureManagement 3.0.0-preview library. [#315](https://github.com/Azure/AppConfiguration-DotnetProvider/issues/315)
+* Removed `IConfigurationRefresher.LoggerFactory` and added success logs for refresh operations. All logs can now additionally be enabled through [these instructions](https://learn.microsoft.com/en-us/dotnet/azure/sdk/logging) for logging with Azure SDK for .NET. [#367](https://github.com/Azure/AppConfiguration-DotnetProvider/pull/367)
+
+* Added the following new API for additional App Configuration geo-replication support. [#385](https://github.com/Azure/AppConfiguration-DotnetProvider/issues/385)
```

Comment:
This is also the first time we are releasing a stable version of the other Connect API for geo replication. I think we should call that out. You could say something like - 
This is the first stable release of the following API introduced in 5.3.0-preview release:
`public AzureAppConfigurationOptions
 Connect(IEnumerable<Uri> endpoints, TokenCredential credential)`